### DISCUSSION TITLE
网站链接的图标在在 index.vue 中怎么引入？直接修改图标名称吗？

### DISCUSSION THREAD

**Discussion by nianshu2022** [imsyy/home#181] [General]
### Site URL

修改了不合适

### Additional Information

![image](https://github.com/imsyy/home/assets/107031519/97f9d58f-678b-469a-be7a-ff246b8433d2)

**Comment by hcymc** [comment]
你从https://xicons.org/的fa标签页选择图标然后复制名称，将名称推荐到siteIcon和import中
